src/bashi/filter_compiler.py

```diff
@@ -198,6 +198,15 @@ def compiler_filter(
             ):
                 reason(output, "hipcc does not support the CUDA backend.")
                 return False
+            # Rule: c19
+            # all ROCm images are Ubuntu 20.04 based or newer
+            # realted to rule d3
+            if UBUNTU in row and row[UBUNTU].version < pkv.parse("20.04"):
+                reason(
+                    output,
+                    "ROCm and also the hipcc  compiler is not available on Ubuntu older than 20.04",
```

Comment:
```suggestion
                    "ROCm and also the hipcc compiler is not available on Ubuntu older than 20.04",
```
One white space to much